packages/teleterm/src/ui/DocumentCluster/ClusterResources/Servers/Servers.tsx

```diff
@@ -60,19 +65,24 @@ function ServerList(props: State) {
 }
 
 const renderConnectCell = (
-  serverUri: string,
-  connect: (serverUri: string) => void
+  getSshLogins: () => string[],
+  onConnect: (login: string) => void
 ) => {
   return (
     <Cell align="right">
-      <ButtonBorder
-        size="small"
-        onClick={() => {
-          connect(serverUri);
+      <MenuSshLogin
+        menuListCss={menuListCss}
```

Comment:
I've been here for just two months, so I suppose @kimlisa might have something more to say. But long-term, injecting arbitrary CSS seems to me like a wrong way of modifying the appearance of a shared component.

Ideally I suppose everything should work OOTB after we change the theme in Teleterm. However, I noticed that the way we use the theme is color-oriented rather than purpose-oriented.

https://github.com/gravitational/webapps/blob/0b65709668092d338ed96909b01052ec6bfcc11c/packages/shared/components/MenuSshLogin/MenuSshLogin.tsx#L130-L137

https://github.com/gravitational/webapps/blob/0b65709668092d338ed96909b01052ec6bfcc11c/packages/design/src/Menu/MenuList.jsx#L32-L33

I might be wrong, but I think this makes it harder to reuse components, because `theme.colors.light` might mean different things in different contexts. I saw [a good thread about that](https://twitter.com/buildsghost/status/1505072846564839424) a couple of days ago which goes into detail about this issue.

OTOH, taking care of that is probably out of scope for this PR.

One solution which comes to my mind is having a special copy of the theme in Teleterm which will override the Teleterm theme in places like this one here where we really need to bend a shared component to make it look a certain way. This means that in this context `theme.colors.light` could be something entirely else than it is in the general Teleterm theme. What do you think about this idea?

Reply:
Thanks for detailed comment, I agree! 
I chose the way with css styling to apply more complicated changes like: different border radius, no separator between items, rounded corners of hovered item.
However, since we have agreed that color customization is sufficient, styling using a provider is simple.